multiapi-engine/src/main/java/com/sngular/api/generator/plugin/asyncapi/util/MapperContentUtil.java

```diff
@@ -270,7 +270,8 @@ private static SchemaFieldObject processFieldObjectList(
     return fieldObject;
   }
 
-  private static void handleItems(final JsonNode schema, Collection<String> modelToBuildList, SchemaFieldObject fieldObject, final boolean required, JsonNode items) {
+  private static void handleItems(
```

Comment:
I think you should need to review your formatter configuration

Reply:
I'm pretty sure the autoformatter is set to use the OSS Style, that's just how it handles this case. Deleting the like break and autoformatting again gives the same result